model/utils.go

```diff
@@ -155,15 +155,20 @@ func NewRandomTeamName() string {
 	return teamName
 }
 
+// NewRandomString returns a random string of the given length.
+// The resulting entropy will be (5 * length) bits.
 func NewRandomString(length int) string {
-	var b bytes.Buffer
-	str := make([]byte, length+8)
-	rand.Read(str)
-	encoder := base32.NewEncoder(encoding, &b)
-	encoder.Write(str)
-	encoder.Close()
-	b.Truncate(length) // removes the '==' padding
-	return b.String()
+	data := make([]byte, 1+(length*5/8))
+	rand.Read(data)
+	return encoding.EncodeToString(data)[:length]
+}
+
+// NewRandomBase32String returns a base32 encoded string of a random slice
+// of bytes of the given size. The resulting entropy will be (8 * size) bits.
+func NewRandomBase32String(size int) string {
```

Comment:
Should we explicitly tell that returning size of the `[]byte` will be in encoded length?

From the comments I understand if I provide `size := 20`, I would get a slice with length of `20`.

Reply:
I actually tried my best to keep the docstring explicit by saying we return a base32 encoded string of a random slice of the given size. But if you understood we return a string of `length == size` then it means it wasn't enough. The final length depends on the input since it may require padding, it will be equal to `base32.StdEncoding.EncodedLen(size)`. 
I can add this information to make it even more clear.

Reply:
I see, it seems like I had a bias of `New***(size int) string` would return  `len(string)==size`. Actually, after reading it more carefully the doc looks OK 👍